code/game/machinery/emancipation.dm

```diff
@@ -0,0 +1,69 @@
+/obj/machinery/emancipation_grill
+	name = "emancipation grill"
+	desc = "Liberator of luggage. Larcenist of belongings."
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "emancipation_grill"
+	density = 0
+	anchored = 1
+	use_power = 1
+	idle_power_usage = 75
+	active_power_usage = 750
+	flow_flags = IMPASSABLE
+	var/list/obj_whitelist = list() //Things that are okay to go through. Frazzle everything else.
+	var/list/obj_blacklist = list() //Things that aren't okay to go through. Don't frazzle everything else.
+
+/obj/machinery/emancipation_grill/New()
+	..()
+	update_icon()
+
+/obj/machinery/emancipation_grill/power_change()
+	. = ..()
+	update_icon()
+
+/obj/machinery/emancipation_grill/update_icon()
+	if(stat & (BROKEN|NOPOWER))
+		icon_state = initial(icon_state)
+	else
+		icon_state = "[initial(icon_state)]_on"
+
+/obj/machinery/emancipation_grill/proc/emancipate(atom/movable/user)
+	if(!user)
+		return
+	if(stat & (BROKEN|NOPOWER))
+		return
+	use_power = 2
```

Comment:
I don't really get what this number does, perhaps add a comment or move it to a define

Reply:
This is so it's using 'active' power.

I have searched and there is NO define for this.